docs/user/checks.rst

```diff
@@ -62,3 +62,40 @@ parameters used for iperf3 checks (e.g. timing, port, username, password,
     may need to update the :doc:`metric configuration
     <device-checks-and-alert-settings>` to enable alerts for the iperf3
     check.
+
+.. _wifi_clients_check:
```

Comment:
I don't love the order of these settings in this page but I'll handle this in a separate commit as I believe you're just following what was done before which in retrospective doesn't seem the best (eg: showing iperf3 settings before other settings which are more important).